ui/imports/shared/views/chat/ProfileContextMenu.qml

```diff
@@ -160,14 +160,14 @@ StatusMenu {
             root.close()
         }
     }
-
+    // TODO Mark as ID verified
     StatusAction {
         id: pendingIdentityAction
         objectName: "pendingIdentity_StatusItem"
         text: isVerificationRequestSent ||
             root.incomingVerificationStatus === Constants.verificationStatus.verified ?
-            qsTr("ID Request Pending....") :
-            qsTr("Respond to ID Request...")
+            qsTr("Review ID verification reply") :
```

Comment:
Not sure about this: initially this pending state meant that a user has sent a verification request and reply has not been received yet

Reply:
Yeah, I will clarify that with Ben later today

Reply:
The whole ID verification flow changed a bit; it seems now you can directly mark someone as "ID verified"

Reply:
I'll probably just revert this part until it's clearer what we should use here; also I don't like the twisted logic here, should be 2 separate menu entries

Reply:
Reverted, pls have another look @MishkaRogachev